package.json

```diff
@@ -143,6 +143,15 @@
           },
           "default": [],
           "description": "Per-workspace list of module directories for the language server to read"
+        },
+        "terraform-ls.rootModules-exclude": {
```

Comment:
I am not sure what the best option name here is, but I would probably try to align the formatting at least more with the existing option, e.g. `excludeRootModules` (without a hyphen).